http/http-advanced-reactive/src/test/java/io/quarkus/ts/http/advanced/reactive/HttpAdvancedReactiveIT.java

```diff
@@ -163,14 +198,146 @@ public void vertxHttpClientRedirection() throws InterruptedException, URISyntaxE
-            assertEquals(HttpStatus.SC_OK, httpStatusCode);
+            assertEquals(SC_OK, httpStatusCode);
             done.countDown();
         });
 
         done.await(TIMEOUT_SEC, TimeUnit.SECONDS);
         assertThat(done.getCount(), equalTo(0L));
     }
 
+    /**
+     * Test must be enabled with property {@link NinetyNineBottlesOfBeerResource#ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_NAME}
+     * set to {@link NinetyNineBottlesOfBeerResource#ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_VAL}.
+     *
+     * @see NinetyNineBottlesOfBeerResource for more information
+     */
+    @EnabledIfSystemProperty(matches = ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_VAL, named = ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_NAME, disabledReason = TEST_REQUIRES_AT_LEAST_2_8_3)
+    @DisabledOnQuarkusVersion(version = DISABLE_IF_NOT_QUARKUS_2_8_3_OR_HIGHER, reason = TEST_REQUIRES_AT_LEAST_2_8_3)
+    @DisplayName("JAX-RS URI path template test")
+    @Test
+    public void uriPathTemplate() {
+        // test parameter name starting with an alphabetic character, containing dash and with literal value
+        req99BottlesOfBeer(1, SC_OK).body(is(NinetyNineBottlesOfBeerResource.FIRST_BOTTLE_RESPONSE));
+        // test parameter name starting with a number and a regex range value
+        req99BottlesOfBeer(2, SC_OK).body(is(NinetyNineBottlesOfBeerResource.SECOND_BOTTLE_RESPONSE));
+        // test parameter name starting with an underscore, containing a dot and regex ranges disjunction
+        req99BottlesOfBeer(3, SC_OK).body(is(String.format(NinetyNineBottlesOfBeerResource.OTHER_BOTTLES_RESPONSE, 3, 3, 2)));
+        // test regex works correctly (matched value between 3 and 99 [inclusive])
+        req99BottlesOfBeer(99, SC_OK)
+                .body(is(String.format(NinetyNineBottlesOfBeerResource.OTHER_BOTTLES_RESPONSE, 99, 99, 98)));
+        // test regex works correctly (no path should be matched)
+        req99BottlesOfBeer(-100, SC_NOT_FOUND);
+    }
+
+    @DisplayName("RESTEasy Multipart Provider test")
+    @Test
+    public void multipartFormDataReader() {
+        app.given()
+                .multiPart(HelloResource.QUERY_PARAM_TITLE, Paths.get("src", "test", "resources", "title.txt").toFile())
+                .formParam(HelloResource.QUERY_PARAM_NAME, "World")
+                .post(HELLO_ENDPOINT)
+                .then().statusCode(SC_OK)
+                .body(CONTENT, is("Hello, World!"))
+                .body(HelloResource.QUERY_PARAM_TITLE, is("Hello Title"));
+    }
+
+    /**
+     * Test must be enabled with property {@link NinetyNineBottlesOfBeerResource#ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_NAME}
+     * set to {@link NinetyNineBottlesOfBeerResource#ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_VAL}.
+     *
+     * @see NinetyNineBottlesOfBeerResource for more information
+     */
+    @EnabledIfSystemProperty(matches = ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_VAL, named = ENABLED_ON_QUARKUS_2_8_3_OR_HIGHER_NAME, disabledReason = TEST_REQUIRES_AT_LEAST_2_8_3)
```

Comment:
This calls `HelloResource`, which is not limited to 2.8.3 or higher, so this annotation (and the comment) can be dropped.